src/js/fc.js

```diff
@@ -538,6 +539,12 @@ var FC = {
             dterm_notch_hz:                 260,
             yaw_lowpass_hz:                 100,
         };
+
+        DEFAULT_PIDS = [
+            42, 60, 35, 20, 70,
```

Comment:
Having defaults in here is somewhat problematic - these will have to reflect whatever is in the firmware, like for example these will have to be updated once betaflight/betaflight#8783 and betaflight/betaflight#8785 have been merged.

Reply:
Yes, it will need to updated, but for now I guess this is the best option. We should have MSP support for default values for 4.2 possibly. That will also fix current need to change filter defaults which have the same issue.

Reply:
Yes, automatically generated messages / client stubs for configuration over MSP would be a nice win, and a worthy target now that we are getting Unified Targets out of the way.